gui/cloudconnectionwizard.cpp

```diff
@@ -620,6 +626,22 @@ void CloudConnectionWizard::handleCommand(CommandSender *sender, uint32 cmd, uin
 		}
 		break;
 
+	case kCloudConnectionWizardLoadCodeCmd: {
+		BrowserDialog browser(_("Select JSON file from ScummVM"), false);
```

Comment:
// I18N: JSON is a file format name
"Select JSON file copied from scummvm.org site"

Reply:
Thanks. I don't know where my mind was when I wrote that. :smile: